src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -28,274 +43,316 @@ function setupTest(
   describe(`visualRefresh=${visualRefresh}`, () => {
     [true, false].forEach(mobile =>
       describe(`mobile=${mobile}`, () => {
-        test(
```

Comment:
this code has only been shifted to inside another loop based on the appLayoutWidget

Reply:
I use this user script which forces hidden whitespace changes by default: https://chromewebstore.google.com/detail/github-whitespace/fnpkdafamnbjoldglihkjjdicofghccm